dsb-integration/src/test/java/com.spectralogic.dsbrowser.integration/tasks/Ds3GetJob_Test.java

```
@@ -55,22 +64,43 @@
+            final SavedSession savedSession = new SavedSession(
+                    TEST_ENV_NAME,
+                    client.getConnectionDetails().getEndpoint(),
+                    "80",
+                    null,
+                    new SavedCredentials(client.getConnectionDetails().getCredentials().getClientId(), client.getConnectionDetails().getCredentials().getKey()),
+                    false,
+                    false);
+            session = new CreateConnectionTask().createConnection(SessionModelService.setSessionModel(savedSession, false), resourceBundle, buildInfoService);
             // pre assuming that file is the direct child of bucket
-            final Ds3TreeTableValueCustom ds3TreeTableValueCustom = new Ds3TreeTableValueCustom(SessionConstants.ALREADY_EXIST_BUCKET, SessionConstants.ALREADY_EXIST_FILES, Ds3TreeTableValue.Type.File, 3718, "2/07/2017 10:28:17", "spectra", false);
-            final ArrayList<Ds3TreeTableValueCustom> listTreeTable = new ArrayList();
+            final Ds3TreeTableValueCustom ds3TreeTableValueCustom = new Ds3TreeTableValueCustom(DS3GETJOB_TEST_BUCKET_NAME,
+                    "SampleFiles.txt",
+                    Ds3TreeTableValue.Type.File,
+                    3718,
+                    "2/07/2017 10:28:17",
+                    "spectra",
+                    false);
+            final ArrayList<Ds3TreeTableValueCustom> listTreeTable = new ArrayList<>();
             listTreeTable.add(ds3TreeTableValueCustom);
-            final ClassLoader classLoader = Ds3GetJob_Test.class.getClassLoader();
-            final URL url = classLoader.getResource(SessionConstants.LOCAL_FOLDER);
             Path path = null;
-            if (url != null) {
-                path = new File(url.getFile()).toPath();
+            try {
+                path = ResourceUtils.loadFileResource("files/");
+            } catch (URISyntaxException | FileNotFoundException e) {
+                e.printStackTrace();
```

If this is a legitimate error condition, then let it fall out to the junit runner instead of handling it here, and swallowing the exception.